x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality_panel/data_quality_summary/summary_actions/check_all/index.test.tsx

```diff
@@ -403,10 +403,10 @@ describe('CheckAll', () => {
 
       // simulate the wall clock advancing
       for (let i = 0; i < totalIndexNames + 1; i++) {
+        await waitFor(() => {});
```

Comment:
This test seem to rely on some very specific internal timings that apperently have changed. I found that adding another waitFor fixes the tests. Feel free to take a look and contribute to make it more robust